pkg/kapp/clusterapply/change_set_view.go

```diff
@@ -48,3 +53,50 @@ func (v *ChangeSetView) Print(ui ui.UI) {
 func (v *ChangeSetView) Summary() string {
 	return v.changesView.Summary() // assumes Print was used before
 }
+
+func (v ChangeSetView) PrintCompleteYamlToBeApplied(ui ui.UI, conf ctlconf.Conf) error {
+	for _, view := range v.changeViews {
+		op := view.ApplyOp()
+		opAndResDesc := fmt.Sprintf("%s: %s", view.ApplyOp(), view.Resource().Description())
+		resYAML := ""
+		switch op {
+		case ClusterChangeApplyOpNoop:
+			continue
+		case ClusterChangeApplyOpDelete:
+			strategy, err := view.ApplyStrategyOp()
+			if err != nil {
+				return err
+			}
+			if strategy == deleteStrategyPlainAnnValue {
+				opAndResDesc = color.RedString("# %s", opAndResDesc)
+			} else {
+				opAndResDesc = color.RedString("# %s %s", strategy, opAndResDesc)
+			}
+		case ClusterChangeApplyOpExists:
+			opAndResDesc = color.GreenString("# %s", opAndResDesc)
+		default:
+			opAndResDesc = color.GreenString("# %s", opAndResDesc)
+			res, err := ctlres.NewResourceWithManagedFields(view.Resource(), false).Resource()
+			if err != nil {
+				return err
+			}
+
+			res, err = diff.NewMaskedResource(res, conf.DiffMaskRules()).Resource()
+			if err != nil {
+				return err
+			}
+
+			resBytes, err := res.AsYAMLBytes()
+			if err != nil {
+				return err
+			}
+			resYAML = string(resBytes)
+
+		}
+		ui.PrintBlock([]byte(fmt.Sprintf(`---
+%s
```

Comment:
Should we mention the update strategies as well?

Reply:
added